deploy/template.yaml

```
@@ -352,6 +352,10 @@ Resources:
                   Environment: !Ref Environment
             - Name: EXTERNAL_WEBSITE_HOST
               Value: !GetAtt ApiGwHttpEndpoint.ApiEndpoint
+            - Name: SESSION_TABLE_NAME
+              Value: !Sub
+                - "cri-kbv-front-sessions-${Environment}"
```

Wrong table name value here, needs to be 
```suggestion
                - "cri-fraud-front-sessions-${Environment}"
```